Code of conduct Update: Steering Council Vote needed (#49)
conda-incubator/governance: CODE_OF_CONDUCT.md

Thank you for helping make this a welcoming, friendly community for all.

> ## [Report a code of conduct incident here.](https://form.jotform.com/221527028480048)

Reviewer: Just wondering who maintains this form, in case we want to improve it in the future?

> "You can enter Anonymous if you wish to report something anonymously. We will keep your personal information confidential even if you provide your contact information."

Can we make the name form input optional instead of having to write "Anonymous" into it?

> "This is optional, but can be very helpful for followup communication. If you are reporting anonymously, then please consider creating and using a random/new email account here."

I'm not sure if this is reasonable, the form should allow submitting anonymously without having to create a new email account.

Reviewer: Alternatively an alias like conduct@conda.org that sends the reports to the CoC Committee members may be useful to set up as well.

Author: @jezdez, some responses:

> Just wondering who maintains this form, in case we want to improve it in the future?

Right now it is owned by condaconduct@gmail.com which is the shared email address for the committee.  That will be set up to distribute any reports to all current committee members.  We were thinking of having the co-chairs have access to it, with a third member being the password recovery option for it.

> Alternatively an alias like [conduct@conda.org](mailto:conduct@conda.org) that sends the reports to the CoC Committee members may be useful to set up as well.

I think that long term a conda.org email would be better, but right now I want to get this out the door.  Also see my general response in this PR.

> Can we make the name form input optional instead of having to write "Anonymous" into it?

Done.

> I'm not sure if this is reasonable, the form should allow submitting anonymously without having to create a new email account.

The form does not require an email address.  The form says this is optional and then asks them to *consider* creating a new email account.  So, I'm not sure what the objection is?

Reviewer: > @jezdez, some responses:
> 
> > Just wondering who maintains this form, in case we want to improve it in the future?
> 
> Right now it is owned by [condaconduct@gmail.com](mailto:condaconduct@gmail.com) which is the shared email address for the committee. That will be set up to distribute any reports to all current committee members. We were thinking of having the co-chairs have access to it, with a third member being the password recovery option for it.
> 
> > Alternatively an alias like [conduct@conda.org](mailto:conduct@conda.org) that sends the reports to the CoC Committee members may be useful to set up as well.
> 
> I think that long term a conda.org email would be better, but right now I want to get this out the door. Also see my general response in this PR.

Fully agreed, and that makes sense. Thank you!

> > Can we make the name form input optional instead of having to write "Anonymous" into it?
> 
> Done.
> 
> > I'm not sure if this is reasonable, the form should allow submitting anonymously without having to create a new email account.
> 
> The form does not require an email address. The form says this is optional and then asks them to _consider_ creating a new email account. So, I'm not sure what the objection is?

I'm mostly trying to put myself in the shoes of the reporting party and wonder if the mere suggestion to create an email account could feel like a barrier for reporting, that's all really. It's not a big worry though.

&nbsp;

And now the longer version...

> # conda Organization Diversity Statement

Reviewer: Hm, I assumed that conda would be capitalized when used at the start of sentence and headlines etc?

Author: Um.... Ask @kathatherine ?

Reviewer 2: Yeah, Jannis is right. I forgot about that capitalization rule when I was doing my initial review of this.

Author: Thanks @kathatherine.  I will update this too.

Reviewer: That reminds me, we need a section in the docs somewhere where we spell this out in detail I think, written language is so awkward sometimes :)

Reviewer 2: Made an issue in conda-docs: https://github.com/conda/conda-docs/issues/803

Went ahead and brought up an issue I've seen for a while when determining how to write about conda. Should try to make a for sure decision about the rules, so we can be consistent and hopefully not confuse everybody!

- Other unethical or unprofessional conduct
- Advocating for, or encouraging, any of the above behaviors

The conda Organization prioritizes marginalized people’s safety over privileged people’s comfort. The conda CoC Committee reserves the right not to act on complaints including, but not limited to:

* ‘Reverse’ -isms, including ‘reverse racism,’ ‘reverse sexism,’ and ‘cisphobia’.
* Reasonable communication of boundaries, such as “leave me alone,” “go away,” or “I’m not discussing this with you.”
* Communicating in a ‘tone’ you don’t find congenial.
* Criticizing racist, sexist, cissexist, or otherwise oppressive behavior or assumptions.

## Behavior Outside of conda Organization Spaces

> The CoC Committee does not influence behavior and membership in spaces outside the conda Organization. However, if you are being harassed by a member of the conda community outside our spaces, we still want to know about it. We will take all good-faith reports of harassment by conda community members seriously. This includes harassment outside our spaces and harassment that took place at any point in time.

Reviewer: ```suggestion
The CoC Committee does not influence behavior and membership in spaces outside the conda Organization. However, if you are being harassed by a member of the conda community outside our spaces, you may still report it to the CoC Committee. We will take all good-faith reports of harassment by conda community members seriously. This includes harassment outside our spaces and harassment that took place at any point in time.
```

To better support situations where immediate action may be required, these guidelines are used *during* live events:

* All conda Organization events will have specific, named Code of Conduct contacts for the events.
* The names and contact mechanisms for the Code of Conduct representatives will be clearly and frequently communicated to event participants.

At conda Organization events, Code of Conduct representatives will attempt to gather and write down [information](#what-to-include-in-a-report) from anyone making a verbal report at a live event. Recording the details in writing is exceedingly important in order for us to effectively respond to reports. If event staff write down a report taken verbally, then the person making the report will be asked to review the written report for accuracy.

For reports made during live events, or in any situation where urgent action is needed:

> * Any two (or more) event organizers, event staff, CoC Committee members or CoC representatives can decide if immediate action is to be taken and what that action is.  In exceptionally dangerous situations, this decision can be made by a single person.  

Reviewer: What's a "CoC representatives"?

Author: I will try and make that clearer.  The goal is to have easy to reach CoC representative at any conda event, even if there aren't CoC members at that event.  We want people in the room who can react quickly.

Reviewer: Ah, so the CoC committee may delegate to one or more representatives for the duration of event, that's smart. Thanks for elaborating!

* ‘Reverse’ -isms, including ‘reverse racism,’ ‘reverse sexism,’ and ‘cisphobia’.
* Reasonable communication of boundaries, such as “leave me alone,” “go away,” or “I’m not discussing this with you.”
* Communicating in a ‘tone’ you don’t find congenial.
* Criticizing racist, sexist, cissexist, or otherwise oppressive behavior or assumptions.

## Behavior Outside of conda Organization Spaces

The CoC Committee does not influence behavior and membership in spaces outside the conda Organization. However, if you are being harassed by a member of the conda community outside our spaces, you may still report it to the CoC Committee. We will take all good-faith reports of harassment by conda community members seriously. This includes harassment outside our spaces and harassment that took place at any point in time.

The CoC Committee reserves the right to exclude people from conda Organization spaces based on their past behavior, including behavior outside conda Organization spaces and behavior towards people who are not in the conda community.  

# Confidentiality and Public Statements to the Community

> The CoC Committee will keep the identity of the reporter confidential. 

Reviewer 3: How does this apply in cases of mandatory reporters serving on the CoC Committee? This seems like a statement that may not be able to be kept.

I am making this comment solely in my personal capacity and am not conveying any rights to any intellectual property of any third parties.

Author: @CJ-Wright, I hadn't thought about mandatory reporters.

**Background:** Some universities (and maybe some companies) have policies that if you are aware of certain bad behaviors by a fellow employee then you must report the behavior to the university (/company). That report might be expected to include the identity of the reporter. If the person being reported works at the same place as a committee member, then that committee member is now in a very hard place.

Does anyone know how common it is for these policies to not allow statements like "This was reported to me in my capacity as an `Organization Name` Code of Conduct committee member, **by a person who wishes to remain anonymous**"?

Reviewer 4: > If the person is uncomfortable, they can seek out another member.

the tricky part of mandatory reporting is that once you, the mandatory reporter, are made aware of a situation you have to report it (the mandatory part). so the person who reports it doesn't really have the opportunity to keep it confidential once you tell a mandatory reporter. at least that's my understanding

Reviewer 5: Fair.

For reports made during live events, or in any situation where urgent action is needed:

* Any two (or more) event organizers, event staff, CoC Committee members or CoC representatives can decide if immediate action is to be taken and what that action is.  In exceptionally dangerous situations, this decision can be made by a single person.  
* These rapid decisions can be reconsidered during the event as more information becomes available.
* The scope of any rapid decision is limited to the current event / situation.
* The report, any related information, and any decisions and consequences will be reported to the full Code of Conduct Committee as soon as possible.

The full Code of Conduct Committee will then consider the report using the full timeline and processes defined below.  The Committee may decide to apply consequences in other spaces beyond the space where the behavior was reported.

> Potential *immediate* consequences for violating the conda Organization Code of Conduct at a live event include:

Reviewer 4: do we need to update this to be "include, but are not limited to"

> Potential *immediate* consequences for violating the conda Organization Code of Conduct at a live event include:

Reviewer 5: ```suggestion
Potential *immediate* consequences for violating the conda Organization Code of Conduct at a live event include, but are not limited to:
```

- Names and contact info, if possible, of **anyone else who witnessed** or was involved in this incident. (Did anyone else observe the incident?)
- **Any other relevant information** you believe we should have

## Anonymous Reporting

> The reporting form supports anonymous incident reporting.  Anonymous reporting works best when the behavior happened in a public space and was witnessed by many.  If an incident is reported anonymously and was not witnessed by others, than the committee may be limited in what actions it can take and what it can report to the larger community.  Nevertheless, we still want to receive these reports.  They are helpful when determining what we need to address as a community, and when looking for evidence of repeated behavior.

Reviewer: ```suggestion
The reporting form supports anonymous incident reporting.  Anonymous reporting works best when the behavior happened in a public space and was witnessed by many.  If an incident is reported anonymously and was not witnessed by others, than the committee may be limited in what actions it can take and what it can report to the larger community.  Nevertheless, the CoC Committee is still interested in receiving these reports.  They are helpful when determining what we need to address as a community, and when looking for evidence of repeated behavior.
```

- Names and contact info, if possible, of **anyone else who witnessed** or was involved in this incident. (Did anyone else observe the incident?)
- **Any other relevant information** you believe we should have

## Anonymous Reporting

> The reporting form supports anonymous incident reporting.  Anonymous reporting works best when the behavior happened in a public space and was witnessed by many.  If an incident is reported anonymously and was not witnessed by others, than the committee may be limited in what actions it can take and what it can report to the larger community.  Nevertheless, the CoC Committee is still interested in receiving these reports.  They are helpful when determining what we need to address as a community, and when looking for evidence of repeated behavior.

Reviewer 6: ```suggestion
The reporting form supports anonymous incident reporting.  Anonymous reporting works best when the behavior happened in a public space and was witnessed by many.  If an incident is reported anonymously and was not witnessed by others, then the committee may be limited in what actions it can take and what it can report to the larger community.  Nevertheless, the CoC Committee is still interested in receiving these reports.  They are helpful when determining what we need to address as a community, and when looking for evidence of repeated behavior.
```

## Conflicts of Interest

> Committee members are expected to recuse themselves in they have a conflict of interest. In addition, the CoC Committee can [vote](#voting-and-decision-making) to remove a committee member from a case, if the committee feels that the member has a conflict of interest.  This [vote](#voting-and-decision-making) requires a simple majority.

Reviewer 3: This may need to be stronger, forcing the recusal of any CoC Committee member named in a case (as reporter or accused). 

I am making this comment solely in my personal capacity and am not conveying any rights to any intellectual property of any third parties.

Author: @CJ-Wright Absolutely, and I think this is uncontroversial.  Look for an update in the next PR.

Author: Done, @CJ-Wright does that work?

| Name | Contact | Employer / Funding | Steering Council Member | Current Term Ends |
| ---- | ---- | ---- | ---- | --- |
| [Dave Clements](https://github.com/tnabtaf) | dcl&#101;ments&#64;anaconda&#46;com, +1 541 914 6324 | [Anaconda](https://anaconda.com/) | | 2023-07-01 |
| [Mahe Iram Khan](https://github.com/ForgottenProgramme) | | [Anaconda](https://anadonda.com/) | | 2023-07-01 | 
| [Ken Odegard](https://github.com/kenodegard) | | [Anaconda](https://anaconda.com/) | | 2023-07-01 |
| [Crystal Soja](https://github.com/csoja), Co-Chair | | [Anaconda](https://anaconda.com/) | Yes | 2023-07-01 |
| [Eric Dill](https://github.com/ericdill), Co-Chair | | [Voltron Data](https://voltrondata.com/) | Yes | 2024-07-01 |
| [Filipe Pires Alvarenga Fernandes](https://github.com/ocefpaf) | ocefpaf@gmail.com | [IOOS/NOAA](https://ioos.noaa.gov/) | Yes | 2024-07-01 |
| [Bianca Henderson](https://github.com/beeankha) | | [Anaconda](https://anaconda.com/) | | 2024-07-01 |
| [Katherine Kinnaman](https://github.com/kathatherine) | kkinnaman.condacoc@gmail.com | [Anaconda](https://anadonda.com/) | | 2024-07-01 | 

> | [](), [External Ombudsperson](#ombudsperson) | | [External Org]() | | 2024-07-01 |

Reviewer: Do we have someone in mind already?

Author: Um, yes, but we won't know for sure until the end of this month.

Reviewer: Gotcha!

# Terms and New Members

* Committee members are appointed for two year terms. Committee members can choose to renew their memberships.
* Committee members can resign before their term ends.
* Committee members can also be removed by a [simple majority vote](#voting-and-decision-making) from their fellow committee members.
* New committee members are added by a simple majority vote as well.

# Eligibility

> Anyone from the community who is interested and able to do CoC Committee work is eligible to be nominated for the committee.  

Reviewer: Is self-nomination allowed?

Author: It should be!  I will make that clearer.

Reviewer: Thank you!
